### ISSUE TITLE
Workflow to reflect typescript changes?

### ISSUE THREAD

**Issue by acherm** [lichess-org/lila-docker#23]
Hi, 

I love this initiative of having a gitpod environment to onboard on lichess dev. 

In the doc it is stated that
```
UI (JS/CSS) development:
To watch for Typescript/SCSS changes and automatically recompile:
docker compose run --rm ui bash -c "/lila/ui/build -w"
``` 

Unfortunately, when I did execute this command, it seems not enough to fully reflect changes on the running website. 
Basically, I have to restart lila with:

```
Scala development:
To restart lila (after making changes to any Scala code):

docker compose restart lila
```

despite my changes are not directly related to Scala code. 

Did I misunderstand something? 

The current dev workflow is not so satisfactory. 
Basically, I was trying to debug the JS code with some console.log... 
And I have to wait lots of time to see the actual effect. 

I'm sure I'm missing something. 

Thanks a lot!

**Comment by fitztrev** [comment]
That should be all that's required. That's the workflow I use.

When you make a change to a TS file, do you see in the ui/build logs that it detected it and recompiled? And then when you refresh, do you do a hard refresh (shift+refresh)?

**Comment by fitztrev** [comment]
#28 is merged as well and will prevent browser caching. If you're still having any issues, just let me know.
